ui/tsconfig.spec.json

```diff
@@ -4,7 +4,8 @@
     "outDir": "./out-tsc/spec",
     "types": ["jest", "node"],
```

Comment:
```suggestion
    "types": ["jest", "node", "cypress", "@testing-library/cypress"],
```

Reply:
Not sure why it keeps showing this change as outdated. @rkennel can you add this change to a commit? Once done, the PR is approved.